### PR TITLE
Added some clarity to the READ.me

### PR DESCRIPTION
**Pull request by paradoxl**
config.py was not ready to intake the .env file. Would be useful to make a note of that in the READ.ME

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,3 @@
+CAPTCHA_PROVIDER = "reCaptcha"
+SITE_KEY = "6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI"
```
Comment (nategraf) [inline .env:2]: Is this your secret key? If so, you should rotate your secrets. Either way, it should not be in this PR.

File: config.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 from os import environ
 from enum import Enum
+from dotenv import load_dotenv
```
Comment (nategraf) [inline config.py:3]: The `dotenv` package is not currently a dependency of this plugin. I'm not sure we should add it as one. If the user does not have `dotenv` installed, this line will currently result in an error and CTFd will not launch.

Comment (nategraf) [reply config.py:3]: Looking at the `dotenv` package, it is not well documented or clearly maintained. I'd prefer not to use it.

https://pypi.org/project/dotenv/

File: config.py [hunk 3]
```diff
@@ -13,6 +14,7 @@ def envvar(key, default=None):
     return default
 
 def config(app):
+    load_dotenv()
```
Comment (nategraf) [inline config.py:17]: I don't want this plugin to assume a `.env` file exists in the plugin directory. CTFd operators will set environment variables in a number of ways, including through Docker or using the `export` command in `bash`. Having this always load the `.env` file could result in confusing behavior for operators.

**Comment by nategraf** [comment]
Thanks for opening a PR! Sorry for being slow to respond. I'll be watching this now, so let me know what you think about my comments.

File: .env (hunk 1 above)
Comment (paradoxl) [reply .env:2]: Apologies, I'm a current student and not as familiar with all the ways that the end goal could have been achieved. I'm also quite new to python and had issues setting env variables. (Hence why i made this suggestion) Those keys are a "Default/testing set" provided by google.

**Comment by nategraf** [comment]
I've thought about this some more and I don't this should be merged in to this repository. Two supported ways to configure these settings are:
1. [Set the environment variables](https://www.schrodinger.com/kb/1842).
2. Edit the `config.py` file.

I don't want to include this `.env` file as a third way because it could be confusing for users. In particular, if the `.env` file is present, it overrides the environment variables set by the user, and so the user would need to delete this file to make configuration via environment variables work.

**Comment by nategraf** [comment]
Thanks for opening a PR though! I definitely want to encourage you to keep doing so.

**Pull request closed without merging**